client/blocks/inline-help/popover.jsx

```diff
@@ -64,6 +67,15 @@ class InlineHelpPopover extends Component {
 		activeSecondaryView: '',
 	};
 
+	componentDidMount() {
+		//Open the Checklist prompt when query string 'onboard=1' is present.
+		const parsed = parse( window.location.href, true );
```

Comment:
I think we should be able to pass this down from the context - using `window` in our components isn't ideal because it means that can only ever be used in the browser, not on the server or on mobile, for example.

Reply:
FWIW a good example can be found here: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/preview/controller.js#L16